modules/pel/peltool/peltool.py

```diff
@@ -358,7 +359,11 @@ def parseAndPrintPELFile(file_path: str, config: Config, exit_on_error: bool) ->
             data = fd.read()
             stream = DataStream(data, byte_order='big', is_signed=False)
             _, json_string = parsePEL(stream, config, exit_on_error)
-            print(json_string)
+            if json_string:
```

Comment:
If we select config.hex option, can we directly print the data in hex format, without parsing it? This could save DataStream and parsePEL function operations.

Reply:
Without parse how will be able check PEL criteria? Is there any way, I hope we will support hex option with other option, right?

Reply:
Oh right!